xtask/src/vm.rs

```diff
@@ -22,7 +22,7 @@ use strum_macros::{Display, EnumIter};
 use crate::internal::*;
 
 #[derive(Debug, Display, Clone, PartialEq, EnumIter)]
-enum Variant {
+pub enum Variant {
     Baremetal,
```

Comment:
pre-existing, but should probably be renamed to Qemu for consistency. I think this is left over from when QEMU was used for UEFI and baremetal.

Reply:
I agree.